hypothesis-python/src/hypothesis/internal/conjecture/utils.py

```diff
@@ -155,9 +155,32 @@ def boolean(data):
     return bool(data.draw_bits(1))
 
 
-def biased_coin(data, p):
+def biased_coin(data, p, forced=None):
```

Comment:
I'd consider making `forced` a keyword-only argument, for much the same reason as we use them in the public API.